Servers/UI/OJS.Servers.Ui/ClientApp/src/common/urls/compose-client-urls.ts

```diff
@@ -0,0 +1,30 @@
+import { CONTESTS_PATH } from './client-urls';
+
+const composeParticipationTypeResultsFullRoute =
+    (id: number, participationType: string) => `/${CONTESTS_PATH}/${id}/${participationType}/results/full`;
+
+const composeContestsWithSelectedCategoryAndStrategyUrl =
+    (categoryId?: number, strategyId?: number) => `/contests?category=${categoryId}&strategy=${strategyId}`;
+
+const composeContestBreadcrumbUrl = (breadcrumbItemId: number) => `/contests?category=${breadcrumbItemId}`;
+
+const composeContestParticipateUrl = (isCompete: boolean, id: number) => {
+    if (isCompete) {
+        return `/contests/${id}/compete`;
+    }
+    return `/contests/${id}/practice`;
```

Comment:
Why do you not use the constant `CONTESTS_PATH`?

Reply:
mb, fixed